.travis.yml

```diff
@@ -1,4 +1,11 @@
 language: java
 jdk: oraclejdk8
 sudo: false
-script: mvn clean deploy
+cache:
+  directories:
+  - $HOME/.m2
+branches:
+  only:
+  - master
+  - /^stable-.*$/
```

Comment:
Do we want to create a separate branch for each release? This will cause a significant increase in the number of branches in our repo. What are the advantages of having a separate branch instead of having a single release branc?

Reply:
Not necessary but i think a nice way to do things.  It's also a pretty standard CI setup and a nice way to allow supporting multiple releases.

Reply:
What is the problem we're trying to solve here, and does it justify having to manage multiple branches?